### PR TITLE
[full-ci] chore: bump test-middleware to v1.8.5

### PR DESCRIPTION
**Pull request by kulmann**
Bumps the `owncloud-test-middleware` to v1.8.4 which adds a retry for ~~async webdav downloads~~ webdav downloads after async uploads.

Edit: now actually bumping to v1.8.5, because v1.8.4 only made one single retry per file. Needs the option for multiple retries.

### CODE REVIEW

**Review by phil-davis — APPROVED** [review]

**Review by kobergj — APPROVED** [review]

**Comment by micbar** [comment]
hmm, upload tests are still failing. Did we miss anything?

**Comment by individual-it** [comment]
strangely it never waits for 1 or 2 retries, ether it passes immediately or never

**Comment by kulmann** [comment]
> strangely it never waits for 1 or 2 retries, ether it passes immediately or never

Yep, seems to be a different issue. @kobergj is already looking into it.

Still good to have the retries on 425 now though. 😅 

@micbar how do you feel about force merging this one? The test failures are unrelated to the PR and it already might help to reduce flakiness.

File: go.mod [hunk 1]
```diff
@@ -291,3 +291,5 @@ require (
 )
 
 replace github.com/cs3org/go-cs3apis => github.com/c0rby/go-cs3apis v0.0.0-20230110100311-5b424f1baa35
+
+replace github.com/cs3org/reva/v2 => github.com/kobergj/reva/v2 v2.0.0-20230303130922-027ec42f8c1d
```
[the remark this answers is not in the document]
Comment (kobergj) [reply go.mod]: Now it's ready

**Comment by kobergj** [comment]
Found out what the issue was. It has been our old friend, the cache. (Who else could it be.) I adjusting caching in reva. Pipeline went green twice already. Should be ready to go now :crossed_fingers:

**Comment by micbar** [comment]
Docker arm32 is not available. Known issue. Merging.

**Pull request merged**